google-datacatalog-postgresql-connector/tests/google/datacatalog_connectors/postgresql/scrape/metadata_scraper_test.py

```diff
@@ -126,6 +127,7 @@ def test_scrape_databases_metadata_on_exception_should_re_raise(
                           connection_args={
                               'database': 'db',
                               'host': 'mysql_host',
+                              'post': 5432,
```

Comment:
Typo on argument name, **post**.

Reply:
Thanks for catching that. Fixed @ cb738f8970e894cc24e40f9fa9a050b39751b9a9